src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({
+  person: {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  },
+}) => (
```

Comment:
it seems that you destruct nested object, it should work like `export const Person = ({name, age, sex, isMarried, partnerName})`